distributed/comm/tests/test_ucx_config.py

```diff
@@ -95,23 +95,26 @@ async def test_ucx_config(cleanup):
 
 
 @pytest.mark.flaky(
-    reruns=10, reruns_delay=5, condition=ucp.get_ucx_version() < (1, 11, 0)
+    reruns=10,
+    reruns_delay=5,
 )
 def test_ucx_config_w_env_var(cleanup, loop):
     env = os.environ.copy()
     env["DASK_RMM__POOL_SIZE"] = "1000.00 MB"
 
     port = "13339"
-    sched_addr = f"ucx://{HOST}:{port}"
+    sched_addr = f"ucx://127.0.0.1:{port}"
```

Comment:
Curious why `HOST` doesn't work for us

Also should we assign to a local `host` variable if this needs to change in the future?

Reply:
It does work, what I noticed is `127.0.0.1` fails less often, that's the only reason I'm changing it here as most other Distributed tests seem to rely on `127.0.0.1` as well, see https://github.com/dask/distributed/blob/main/distributed/cli/tests/test_dask_scheduler.py for instance. I think we don't need to worry about a `host` variable here honestly, being straightforward seems good enough.

Reply:
Ok maybe we should leave a comment about that

Reply:
Done in https://github.com/dask/distributed/pull/5765/commits/3f0f589a070ee693d563c5e006adef354789a80d, I expanded a bit your comment suggestion too.